lib/node_modules/@stdlib/math/base/ops/cneg/lib/main.js

```diff
@@ -20,45 +20,69 @@
 
 // MODULES //
 
-var negate = require( './cnegate.js' );
+var real = require( '@stdlib/complex/real' );
+var imag = require( '@stdlib/complex/imag' );
+var Complex128 = require( '@stdlib/complex/float64' );
 
 
 // MAIN //
 
 /**
-* Negates a complex number.
+* Negates a double-precision complex floating-point number.
 *
-* @param {(Array|TypedArray|Object)} [out] - output array
-* @param {number} re - real component
-* @param {number} im - imaginary component
-* @returns {(Array|TypedArray|Object)} negated components
+* @param {Complex128} z - complex number
+* @returns {Complex128} result
 *
 * @example
-* var v = cneg( -4.2, 5.5 );
-* // returns [ 4.2, -5.5 ]
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var real = require( '@stdlib/complex/real' );
+* var imag = require( '@stdlib/complex/imag' );
+* var cneg = require( '@stdlib/math/base/ops/cneg' );
```

Comment:
```suggestion
```

We don't explicitly require the package in implementation JSDoc examples. Only in `lib/index.js`.